nixos/doc/manual/release-notes/rl-2305.section.md

```diff
@@ -144,6 +144,9 @@ In addition to numerous new and upgraded packages, this release has the followin
 
 - conntrack helper autodetection has been removed from kernels 6.0 and up upstream, and an assertion was added to ensure things don't silently stop working. Migrate your configuration to assign helpers explicitly or use an older LTS kernel branch as a temporary workaround.
 
+- The `services.pipewire.config` options have been removed, as they have basically never worked correctly. All behavior defined by the default configuration can be overridden with drop-in files as necessary.
```

Comment:
Where to place those drop-in files? That's a bit unclear.

Reply:
I'm not sure how verbose we want to be here. The drop-in files go into `/etc/pipewire/<whatever>.conf.d` and there are multiple config files, so it's hard to explain without getting into a lot of detail. Also I kinda just hope no one is using this, because it does not work the way anyone would expect.

Reply:
Given that there's [a whole wiki page dedicated to tweaking `services.pipewire.config`](https://nixos.wiki/wiki/PipeWire), I think going into detail is warranted.

Reply:
You'd probably even want to announce it in the breaking changes thread.

Reply:
That entire page will need to be updated. It's on my list. I'll probably write up a post detailing the migration steps and post that in the thread.

Reply:
Draft migration guide: https://gist.github.com/K900/20fb1651d72411e3419d278bb29343ea

Reply:
The migration guide should preferrably be mentioned in the release notes too, since it's a pretty big chunk of not necessarily obvious manual work.

Reply:
It's definitely going in the docs somewhere, I just need to find a good place to put it in.